gateway/tests/e2e/feedback.rs

```diff
@@ -4,14 +4,18 @@ use serde_json::{json, Value};
 use tokio::time::{sleep, Duration};
```

Comment:
Make it so that these tests add two tags to the feedback to ensure that it works as expected.